ingestion/src/metadata/data_quality/validations/base_test_handler.py

```diff
@@ -65,37 +66,18 @@ def run_validation(self) -> TestCaseResult:
         """
         raise NotImplementedError
 
+    @staticmethod
     def get_test_case_param_value(
-        self,
-        test_case_param_vals: list[TestCaseParameterValue],
+        test_case_param_vals: List[TestCaseParameterValue],
         name: str,
         type_: T,
         default: Optional[R] = None,
         pre_processor: Optional[Callable] = None,
     ) -> Optional[Union[R, T]]:
-        """Give a column and a type return the value with the appropriate type casting for the
```

Comment:
Should we keep this doc string?

Reply:
I moved it to the underlying utils library bc this doesnt have much meat. didnt want to keep redundant material. wdyt?

Reply:
here: https://github.com/open-metadata/OpenMetadata/pull/18115/files#diff-81b87512f0e5a37568c837e4c3631ae91431312ff2b185e864aa18a260d1c8faR20